src/Components/Home.js

```diff
@@ -2,13 +2,18 @@ import "../CSS/Home.css";
 import SWECCmap from "../Data/img/location.png";
 import UWlogo from "../Data/img/UW_logo.png";
 import { InstagramEmbed } from "react-social-media-embed";
+import { Posts } from "./Posts.js";
 
 function HomePage() {
   const meetingInfo = (
     <div className="meetingsSection">
       <div className="meetingsInfoSection">
         <h2 className="sweccMeetings">SWECC Meetings</h2>
-        <p className="meetingInfo">Held in the EAC, these meeting feature topics in a vast number of areas, including professional development, resume building, and mentor circles</p>
+        <p className="meetingInfo">
+          Held in the EAC, these meeting feature topics in a vast number of
```

Comment:
Careful touching more code than necessary. Obvious you were just formatting though so it shouldn't matter too much